content/en/docs/v3.4/get-started/etcd.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Get Started with etcd
+weight: 910
+description: Getting started with etcd on Linux or macOS
+---
+
+# Quickstart
+
+This guide shows how to download, setup, and test a simple single-member cluster of etcd.
+
+## Before you start
+
+Before you begin, this etcd guide requires the following:
+
+* Linux or macOS
+
+## Get started
+
+## Step 1 - Download etcd
+
+Set the environment variables to help with the commands that follow.
+
+```
+ETCD_VER=v3.4.15
+
+# choose either URL
+GOOGLE_URL=https://storage.googleapis.com/etcd
+GITHUB_URL=https://github.com/etcd-io/etcd/releases/download
+
+DOWNLOAD_URL=${GITHUB_URL}
+```
+
+### Download
+
+On Linux
+```
+rm -f /tmp/etcd-${ETCD_VER}-linux-amd64.tar.gz
+rm -rf /tmp/test-etcd && mkdir -p /tmp/test-etcd
+
+curl -L ${DOWNLOAD_URL}/${ETCD_VER}/etcd-${ETCD_VER}-linux-amd64.tar.gz -o /tmp/etcd-${ETCD_VER}-linux-amd64.tar.gz
+tar xzvf /tmp/etcd-${ETCD_VER}-linux-amd64.tar.gz -C /tmp/test-etcd --strip-components=1
+```
+
+On macOS
+```
+rm -f /tmp/etcd-${ETCD_VER}-darwin-amd64.zip
+rm -rf /tmp/test-etcd && mkdir -p /tmp/test-etcd
+
+curl -L ${DOWNLOAD_URL}/${ETCD_VER}/etcd-${ETCD_VER}-darwin-amd64.zip -o /tmp/etcd-${ETCD_VER}-darwin-amd64.zip
+unzip /tmp/etcd-${ETCD_VER}-darwin-amd64.zip -d /tmp
+mv /tmp/etcd-${ETCD_VER}-darwin-amd64/* /tmp/test-etcd
+```
+
+### Check the binaries
+
+Check that both etcd and etcdctl are correct.
+<!-- TODO: find out if there is a checksum we can use to verify the downloads -->
+
+```
+/tmp/test-etcd/etcd --version
+ETCDCTL_API=3 /tmp/test-etcd/etcdctl version
+```
+
+## Step 2 - Move etcd
+
+etcd can be run from any directory, so we can move the binary to a location in the system PATH.
+
+```
+mv /tmp/test-etcd/etcd /usr/local/bin/
+mv /tmp/test-etcd/etcdctl /usr/local/bin/
+```
+
+## Step 3 - Run etcd
+
+Now that the binaries are in place, here is a simple test that everything is working
+```
+# run etcd
+etcd
+
+# set a key
+etcdctl put mykey "this is awesome"
+
+# retreive key
+etcdctl get mykey
+```
```

Comment:
It is generally better to avoid putting prose inside command-sequence blocks.

Reply:
Besides, the commands above are the crux of the getting-started instructions. They need more supporting prose and details.

For example, it should be clear that you need to run etcd and etcdctl from two different shells (or, run etcd in the background, but I wouldn't suggest that).

Also, you should format commands as terminal/console "code" blocks, with each command being preceded by a `$` prompt. This allows you to include some command output -- which in turn helps readers validate that their commands are working as expected.

Reply:
> Besides, the commands above are the crux of the getting-started instructions. They need more supporting prose and details.
> 
> For example, it should be clear that you need to run etcd and etcdctl from two different shells (or, run etcd in the background, but I wouldn't suggest that).

I agree, I'll  rework this bit.

> Also, you should format commands as terminal/console "code" blocks, with each command being preceded by a `$` prompt. This allows you to include some command output -- which in turn helps readers validate that their commands are working as expected.

I tend to agree with this -- i was going back and forth about including the `$`, but I think i will in this case. I also agree that adding the output would be helpful here.